x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_type_form.test.tsx

```diff
@@ -363,6 +363,110 @@ describe('action_type_form', () => {
       ],
     ]);
   });
+
+  describe('Security Solution', () => {
```

Comment:
Yeah seeing a test that's explicitly called "security solution" within a plugin that isn't the security solution gives me serious pause.